pkg/question/question_test.go

```diff
@@ -71,18 +72,18 @@ func TestAskQuestion_StringOptionAnswer(t *testing.T) {
 	const testResponse = cli.ResponseNo
 	initQuestionTest(t, testResponse+"\n")
 
-	answer := question.AskQuestion(input)
-	output := cleanupQuestionTest()
-	th.Equals(t, expectedOutput, output)
-	th.Equals(t, testResponse, answer)
+	// answer := question.AskQuestion(input)
+	// output := cleanupQuestionTest()
+	// th.Equals(t, expectedOutput, output)
+	// th.Equals(t, testResponse, answer)
```

Comment:
Rather than commenting out the parts of the test that fail, we could write a function that skips the test when running on GitHub Actions (which sets an environment variable `CI` to `true`). That way, we are explicit about the fact that we're skipping the tests for now because we don't know how to get Bubble Tea to run successfully under a test, but that we eventually need to fix it before we go to prod.

Here's basically what I'm thinking, [based loosely on this](https://stackoverflow.com/questions/24030059/skip-some-tests-with-go-test#50631395):

```golang
func skipBubbleTeaTestOnCI(t *testing.T) {
    if os.Getenv("CI") != "" {
        t.Skip("Skipping this tests until we decide how to run Bubble Tea within a test")
    }
}

// Then, at the top of each test that fails because of Bubble Tea, add this line:
skipBubbleTeaTestOnCI(t)
```

Reply:
For future reference, this strategy may be appropriate for the remaining question-testing functions in this file, which are still failing. But these particular tests are now irrelevant because the `AskQuestion` function was removed.